### ISSUE TITLE
RTU Device not supporting function 06

### ISSUE THREAD

**Issue by SteveOss** [favalex/modbus-cli#22]
I have a device that only supports writing to multi holding registers (function 16) and not single register write (function 06). So cannot use the CLI (as far as I can see) to write a single register. Think there needs to be some option to implement a single
register write using function 16.

**Comment by favalex** [comment]
Yeah, I agree, there should be an option to force using function 16 even when writing just one register (and probably same for coils.)

Adding it to my TODO
